src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -231,9 +236,8 @@ public with sharing class PMT_PaymentWizard_CTRL {
     /*******************************************************************************************************
     * @description ActionMethod for the Calculate button that given the current opp outstanding balance, 
     * creates the specified new payments to cover the amount.
-    * @return PageReference null
     */ 
-    public pageReference calculate() {
+    public void calculate() {
```

Comment:
I'm not finding specifics in the docs around it, but it does state PageReferences should be used for:
To view or set query string parameters and values for a page
To navigate the user to a different page as the result of an action method